
Header header

geometry_msgs/Vector3 points
geometry_msgs/Vector3[] arraypoints


geometry_msgs/Vector3 points2
geometry_msgs/Vector3[] arraypoints2

int64 size
int64[] allsize

sensor_msgs/Image image

geometry_msgs/Vector3 pixelpoints
geometry_msgs/Vector3[] arraypixelpoints
